appveyor.yml

```diff
@@ -26,6 +32,7 @@ install:
   - node --version
   - npm --version
   - npm i
+  - mv deltachat-core-rust/target _target
```

Comment:
After `npm i` we have built everything and need to make sure to update `_target` so it's up to date before being cached again.